.codeclimate.yml

```diff
@@ -0,0 +1,73 @@
+# skip irrelevant directories
+exclude_paths:
+    - "**/vendor/"
+    - "build/"
+    - "doc/"
+    - "bin/"
+    - "data/"
+    - "var/"
+    - "eZ/Bundle/EzPublishCoreBundle/Tests/"
+    - "eZ/Bundle/EzPublishDebugBundle/Tests/"
+    - "eZ/Bundle/EzPublishIOBundle/Tests/"
+    - "eZ/Bundle/EzPublishRestBundle/Tests/"
+    - "eZ/Publish/API/Repository/Tests/"
+    - "eZ/Publish/Core/Base/Tests/"
+    - "eZ/Publish/Core/FieldType/Tests/"
+    - "eZ/Publish/Core/Helper/Tests/"
+    - "eZ/Publish/Core/IO/Tests/"
+    - "eZ/Publish/Core/Limitation/Tests/"
+    - "eZ/Publish/Core/MVC/Symfony/Cache/Tests/"
+    - "eZ/Publish/Core/MVC/Symfony/Controller/Tests/"
+    - "eZ/Publish/Core/MVC/Symfony/Event/Tests/"
+    - "eZ/Publish/Core/MVC/Symfony/EventListener/Tests/"
+    - "eZ/Publish/Core/MVC/Symfony/FieldType/Tests/"
+    - "eZ/Publish/Core/MVC/Symfony/Locale/Tests/"
+    - "eZ/Publish/Core/MVC/Symfony/Matcher/Tests/"
+    - "eZ/Publish/Core/MVC/Symfony/Routing/Tests/"
+    - "eZ/Publish/Core/MVC/Symfony/Security/Tests/"
+    - "eZ/Publish/Core/MVC/Symfony/SiteAccess/Tests/"
+    - "eZ/Publish/Core/MVC/Symfony/Templating/Tests/"
+    - "eZ/Publish/Core/MVC/Symfony/View/Tests/"
+    - "eZ/Publish/Core/Pagination/Tests/"
+    - "eZ/Publish/Core/Persistence/Tests/"
+    - "eZ/Publish/Core/Persistence/Cache/Tests/"
+    - "eZ/Publish/Core/Persistence/Doctrine/Tests/"
+    - "eZ/Publish/Core/Persistence/Legacy/Tests/"
+    - "eZ/Publish/Core/Repository/Tests/"
+    - "eZ/Publish/Core/REST/Client/Tests/"
+    - "eZ/Publish/Core/REST/Common/Tests/"
+    - "eZ/Publish/Core/REST/Server/Tests/"
+    - "eZ/Publish/Core/Search/Tests/"
+    - "eZ/Publish/Core/Search/Legacy/Tests/"
+    - "eZ/Publish/Core/SignalSlot/Tests/"
+    - "eZ/Publish/SPI/Tests/"
```

Comment:
can these user pattern matches instead for Tests folders?

Reply:
@andrerom not sure what you mean

Reply:
Like here: https://docs.codeclimate.com/v1.0/docs/example-codeclimateyml

Reply:
As in can we match something like "**/Tests/" ?